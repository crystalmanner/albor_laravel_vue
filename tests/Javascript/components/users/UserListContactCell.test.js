import { mount } from '@vue/test-utils'
import { FIXTURE_USER } from 'tests/__data__/user'
import Component from '~/components/users/UserListContactCell.vue'

describe('User List Contact Cell component', () => {
  describe('Visuals', () => {
    test('deafult', () => {
      const wrapper = mount(Component)
      expect(wrapper.element).toMatchSnapshot()
    })
    test('basic', () => {
      let item = FIXTURE_USER
      const wrapper = mount(Component, { propsData: { item } })
      expect(wrapper.element).toMatchSnapshot()
      expect(wrapper.text()).toContain(FIXTURE_USER.first_name)
      expect(wrapper.text()).toContain(FIXTURE_USER.last_name)
      expect(wrapper.text()).toContain(FIXTURE_USER.email)
    })
  })
})
