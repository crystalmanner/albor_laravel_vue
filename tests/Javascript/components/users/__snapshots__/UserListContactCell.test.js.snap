// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User List Contact Cell component Visuals basic 1`] = `
<div>
  
  Demo Admin
  
  <br />
   
  <small>
    demoAdmin@example.com
  </small>
</div>
`;

exports[`User List Contact Cell component Visuals deafult 1`] = `
<div>
  
   
  
  <br />
   
  <small>
    
  </small>
</div>
`;
